scripts/README.md

```diff
@@ -0,0 +1,66 @@
+## Development Scripts Readme
+
+This directory contains scripts to perform testing and maintainence tasks for developers of the StellarGraph  library. These scripts should not be needed for users of the library.
+
+Currently there are two scripts, one to formats and clean the specified Juptyer notebooks, the other to run specific demo notebooks and check if there are any errors.  In the future the functionality of these two scripts may be combined.
+
+
+### Format and clean up demo Jupyter notebooks
+
+The script `format_notebooks.py` uses `nbconvert` to process all code and output cells in the specified locations optionally remove Tensorflow warnings and stderr outputs, format and number the code cells, and set the kernel to the default of "Python 3".
```

Comment:
@adocherty Do we need a requirements.txt for the scripts?

When I try to run the following command,

> python scripts/format_notebooks.py -h

I get the following error,

```
Traceback (most recent call last):
  File "scripts/format_notebooks.py", line 32, in <module>
    from black import format_str, FileMode, InvalidInput
ModuleNotFoundError: No module named 'black'
```

Clearly, I can install black to get around this problem, but what about the version. Also, do we need to install nbconvert (is this another python library?)

I had a look at the StellarGraph requirements.txt and the above are not included.

Reply:
I've added a requirements.txt file containing black, nbconvert, (part of and installed by Jupyter so should be covered, but just in case) and treon (for the other demo script).  Added a requirements section in the readme.